green_cli/green.py

```diff
@@ -260,6 +286,24 @@ def register(session):
     """Register an existing wallet"""
     return context.authenticator.register(session.session_obj)
 
+@green.command()
+@no_warn_sysmsg
+@with_login
+def getsystemmessages(session):
+    """Get unread system messages"""
+    while True:
+        message = gdk.get_system_message(session.session_obj)
+        if not message:
+            break
+
+        click.echo("--- MESSAGE STARTS ---")
+        click.echo(message)
+        click.echo("--- MESSAGE ENDS ---")
+        if not click.confirm("Mark message as read (sign and send acknowledgement to the server)?"):
+            break
+
+        _gdk_resolve(gdk.ack_system_message(session.session_obj, message))
```

Comment:
Can this just be part of the if if you remove the `not`?

Reply:
In the case of a negative response I also want to completely break out of the loop